Add SNES9x to RAPPS

LothaireCliquennois opened the pull request:
[SNES9x.txt](https://github.com/reactos/rapps-db/files/3467778/SNES9x.txt)

This is my second ever pull request on ROS's repos, tell me if anything goes bad.
Tested it in my fresh VM this morning.

katahiromz commented:
Screenshot please.

shytzedaka commented:
Is it functional? Does it run the games? 
Whats the status on Audio? is it Unknown?

LothaireCliquennois commented:
> Is it functional? Does it run the games?
> Whats the status on Audio? is it Unknown?

It is functional. I have no problem playing games...
Sound is another story, trying to get Virtualbox's sound working.
Might be good to have the built-in media player to support mp3 or ogg files for testing.

LothaireCliquennois commented:
> Screenshot please.

![image](https://user-images.githubusercontent.com/41753021/62477891-2c799680-b7aa-11e9-8296-87fcfe7664b1.png)

katahiromz commented:
```txt
Version = 1.6.0
```
Version is not 1.6.0 but 1.60.

BenNottelling commented:
> OpenGL was the only change I made to the settings...

Is that needed to have it run?
If so, please add it to the description when addressing the comment by @katahiromz

LothaireCliquennois commented:
> > OpenGL was the only change I made to the settings...
> 
> Is that needed to have it run?
> If so, please add it to the description when addressing the comment by @katahiromz

It is needed to run, yes.
I adressed the version issue, and I adressed the description to add to the .txt file
[SNES9x.txt](https://github.com/reactos/rapps-db/files/3471280/SNES9x.txt)

katahiromz commented on SNES9x.txt:
```diff
@@ -0,0 +1,16 @@
+[Section]
+Name = SNES9x
+Version = 1.6.0
```
```suggestion
Version = 1.60
```

katahiromz commented:
Can you add notice about OpenGL? Thanks.

LothaireCliquennois commented:
[SNES9x.txt](https://github.com/reactos/rapps-db/files/3478634/SNES9x.txt)
I made a file with all the changes, and sent it yesterday but it looks like I'm too dumb to do a freaking commit, even though it sounds like the first thing to learn in the git workflow.

LothaireCliquennois commented:
> Now it should be good to go.

And I am a freaking genius lmao I learned how to commit changes to my pull request...
That might seem retarded, but it's a good thing...

katahiromz commented on SNES9x.txt:
```diff
@@ -0,0 +1,18 @@
+License = GPL
+Description = a portable, freeware Super Nintendo Entertainment System (SNES) emulator. You'll need to change the video renderer to OpenGL in the video settings or in the ini file.
+Size = 2.61 MiB
+Size = 2.61 MiB
```
You doubled this line.

katahiromz commented on SNES9x.txt:
```diff
@@ -0,0 +1,18 @@
+[Section.040c]
+Description = Un émulateur portable et gratuit de Super Nintendo Entertainment System (SNES). Vous devrez utiliser OpenGL comme moteur de rendu dans les paramètres vidéo ou dans le fichier .ini.
+[Section.0a]
+[Section.0a]
```
Ditto.
